drop/modules/aberrant-splicing-pipeline/FRASER/Summary.R

```diff
@@ -93,15 +93,12 @@ for(type in psiTypes){
   after
 }
 
-#' # Results
+#' ## Results
 res <- fread(snakemake@input$results)
 file <- gsub(".html$", ".tsv", snakemake@output$wBhtml)
```

Comment:
Same as for aberrant expression, please specify this path in the Snakemake header